doc/mgr/dashboard.rst

```diff
@@ -236,6 +236,11 @@ exist and you may find yourself in the situation that you have to use them::
   $ ceph dashboard set-rgw-api-admin-resource <admin_resource>
   $ ceph dashboard set-rgw-api-user-id <user_id>
 
+If you are using a self-signed certificate in your Object Gateway setup, then
+you should disable SSL verification in the dashboard::
```

Comment:
s/SSL verification/certificate verification/ maybe?
And maybe explain why it should be disabled? To avoid errors?